Improve logging

scottqueen-bixal opened the pull request:
Fixes #772 

When running `cron.js` the spawned scripts executed were outputing logging was not making its way up to the cloud log stream. 

This works to update `winston` for better reporting syntax and includes a new function to handle logger messages, specifically if we have an error to log it as an error.

levinmr commented:
@scottqueen-bixal I like your changes.  I went a little farther with it and moved the logger config to a single place, since the app was creating loggers in several different files.  Also added an optional env var to set the log level.

scottqueen-bixal commented:
> @scottqueen-bixal I like your changes. I went a little farther with it and moved the logger config to a single place, since the app was creating loggers in several different files. Also added an optional env var to set the log level. Take a look and see what you think

Excellent, with test and all! - This looks good to me if we want to push up and see how it goes on cloud log stream?
